test/unit/pcv/NamedStaticPCVDepositWrapper.test.ts

```diff
@@ -0,0 +1,296 @@
+import { getCore, getAddresses, expectRevert, expectEvent } from '../../helpers';
+import { expect } from 'chai';
+import hre, { ethers } from 'hardhat';
+import { Signer } from 'ethers';
+import { Core, Fei, NamedStaticPCVDepositWrapper } from '@custom-types/contracts';
+const toBN = ethers.BigNumber.from;
+
+describe('NamedStaticPCVDepositWrapper', function () {
+  const impersonatedSigners: { [key: string]: Signer } = {};
+
+  let underlyingTokenAmount: string;
+  let governorAddress: string;
+  let newDepositName: string;
+  let balance: string;
+  let fei: string;
+  let core: Core;
+  let deposit: NamedStaticPCVDepositWrapper;
+
+  before(async () => {
+    const addresses = await getAddresses();
+
+    // add any addresses you want to impersonate here
+    const impersonatedAddresses = [
+      addresses.userAddress,
+      addresses.pcvControllerAddress,
+      addresses.governorAddress,
+      addresses.pcvControllerAddress,
+      addresses.minterAddress,
+      addresses.burnerAddress,
+      addresses.beneficiaryAddress1,
+      addresses.beneficiaryAddress2
+    ];
+
+    for (const address of impersonatedAddresses) {
+      await hre.network.provider.request({
+        method: 'hardhat_impersonateAccount',
+        params: [address]
+      });
+
+      impersonatedSigners[address] = await ethers.getSigner(address);
```

Comment:
use getImpersonatedSigner helper method NOTE: also needs to be imported from helpers

```suggestion

      impersonatedSigners[address] = await getImpersonatedSigner(address);
```